nox/virtualenv.py

```diff
@@ -122,9 +122,11 @@ def _resolved_interpreter(self):
 
         Based heavily on tox's implementation (tox/interpreters.py).
         """
+        from nox.sessions import _SessionSkip
```

Comment:
@theacodes I don't like this. It was to avoid a circular import since `nox.sessions` imports `nox.virtualenv`. What do you think we should do?

Reply:
Why not just raise a new `InterpreterNotFound` error? We could use that as a more general purpose solution to #93.

Reply:
Care to take the wheel here? I'm not really sure where to put it / what other corners to test it in.